xendit-android/src/main/java/com/xendit/Xendit.java

```diff
@@ -201,10 +201,10 @@ public static boolean isCvnValid(String creditCardCVN) {
      * @param tokenCallback The callback that will be called when the token creation completes or
      *                      fails
      */
-    public void createSingleUseToken(final Card card, final int amount, final TokenCallback tokenCallback) {
+    public void createSingleUseToken(final Card card, final int amount, final String onBehalfOf, final TokenCallback tokenCallback) {
```

Comment:
we need to preserve the original format as well, otherwise all user (not just xenplatform user) will need to update their code when they're updating the SDK

Reply:
do it for all the public methods

Reply:
so, you mean remove onBehalfOf in public methods?

Reply:
no, i'm saying we should create a new public method instead of overriding the existing one

Reply:
i see